validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/BlobsetSequences.java

```diff
@@ -99,4 +102,47 @@ public void endpoint_config_connection_success_reconnect() {
           && stateStatus.level == Level.NOTICE.value();
     });
   }
+
+  @Test
+  @Description("Restart and connect to same endpoint and expect it returns.")
+  public void endpoint_config_connection_success_restart() {
```

Comment:
this test seems misnamed because it has nothing to do with endpoint config -- it's purely restart. As per the discussion in the doc, we're really looking at two separate tests for "restart" -- one it just "does the system restart" and shouldn't be tied to anything to do with endpoint, and the other is "does endpoint redirect survive restart" -- which would be some extra logic on top of this one.  Anyway, I would suggest just renaming/redescribing this test to the simpler "restart only" one, and then we can add the other one later.

Reply:
So rename it to "_restart_only" ?

Reply:
Yes I meant endpoint_config_connection_success_restart_only.

Reply:
If it's system_mode_restart is there somewhere else to put the test?